ga4gh/server/cli/__init__.py

```diff
@@ -9,14 +9,6 @@
 
 import ga4gh.schemas.protocol as protocol
 
-# the maximum value of a long type in avro = 2**63 - 1
-# (64 bit signed integer)
-# http://avro.apache.org/docs/1.7.7/spec.html#schema_primitive
-# TODO in the meantime, this is the max value pysam can handle
-# This should be removed once pysam input sanitisation has been
-# implemented.
-AVRO_LONG_MAX = 2**31 - 1
```

Comment:
Protobuf's JSON serialization has a similar problem it works around, in that it doesn't serialize longs as Javascript numbers (that max at 2**57 -1 or something), but as strings.